docs/pt/developer-docs/am-i-ready-for-review.md

```diff
@@ -0,0 +1,212 @@
+   - Se um jogo tem múltiplos modos de jogabilidade, você pode querer exibir dados completamente diferentes por modo. Condicionais permitem que você faça isso.
+
+### Você testou seu Rich Presence?
+
+Certifique-se de testá-lo em todos os contextos do jogo (menus, título, no jogo, em diferentes modos de jogo, início de jogo, fim de jogo) para ver se alguma vez ele exibe algo estranho.
+
+## Alguma das suas condições usa um único endereço (ou até mesmo uma única condição total)?
+
+Ter um único endereço pode tornar sua conquista propensa a gatilhos falsos, especialmente se um emulador inicializa a memória de forma estranha, ou se a memória tem alguma oscilação. Como resultado, queremos evitar lógica excessivamente simples.
+
+- **Sempre** tenha mais de uma condição. Encontre um estado válido do jogo para combinar com a lógica, como "No jogo" ou "jogador tem controle". Você quer ter certeza de que o jogador está realmente em um ponto onde as outras condições são válidas.
+- Se você tem mais de uma condição, mas apenas um endereço está sendo usado, fique atento às transições *"ligado para desligado"*. `Delta < X` seguido por `Mem >= X` é efetivamente ir de "ligado para desligado" com X sendo tratado como o limiar.
+  - Encontre algumas outras condições para adicionar a isso para tornar esse tipo de coisa mais robusto e garantir que o jogador esteja realmente em um bom estado do jogo quando elas acontecerem.
+
+## Você está usando Hits Armazenados (às vezes chamados de Hits Alvo)? Você tem uma maneira de Resetar esses hits armazenados?
+
+### O que são e não são Hits Armazenados
+
+**Hits armazenados são**: quando você especifica uma quantidade específica de hits em uma condição (ou cadeia) e a condição não é considerada verdadeira até que essa quantidade alvo de hits tenha sido atingida.
+
+**Hits armazenados não são**: o número rolante de hits no editor de conquistas que aparece em condições sem um alvo. Esses são para rastrear quantos frames uma condição já foi verdadeira e são puramente informativos.
+
+- Hits armazenados devem sempre ter uma maneira de serem resetados, como quando um desafio falha, quando um jogador não está mais no nível relevante, ou o jogador sofre um game over. Caso contrário, esses hits podem permanecer e causar um gatilho falso no contexto errado.
+- Solução: Adicione uma condição ou cadeia de condições (como Com And Próximo) marcada com a flag Reset If. Quando a(s) condição(ões) for(em) verdadeira(s), os hits armazenados serão limpos.
+
+### Você está usando hits armazenados para ajudar a verificar quando um valor muda de uma coisa para outra?
+
+Isso é algo que você vê em conjuntos antigos de quando o kit de ferramentas era menor. Isso é quase sempre a maneira errada de fazer isso.
+
+- Solução**: Use a flag Delta!
```

Comment:
May be missing an accompanying `**`?

Reply:
That is also wrong in the original version (in English). I'll change in both files.